builder/dockerfile/dispatchers.go

```diff
@@ -651,6 +638,17 @@ func arg(b *Builder, args []string, attributes map[string]bool, original string)
 	return b.commit("", b.runConfig.Cmd, fmt.Sprintf("ARG %s", arg))
 }
 
+// SHELL
+//
+// Set the shell used for RUN commands.
+func shell(b *Builder, args []string, attributes map[string]bool, original string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("SHELL requires exactly one argument definition")
+	}
+	b.runConfig.Shell = strings.Split(args[0], " ")
```

Comment:
Actually, scrub that. It won't be in the next update as (per @tianon) it's JSON only.